### PR TITLE
Create ethical_statement.md

### PR DESCRIPTION
**Pull request by munsterlander**
First draft.  It is a light rip from LPI, although I do feel enough has changed that we are good.  Additionally, as we add and take away, it will move further away from the source document.

### CODE REVIEW

**Comment by munsterlander** [comment]
One thing to note here, I was just browsing and didn't realize that Github has a community section: https://github.com/opendatacam/opendatacam/community where a Code of Conduct based on an open license can be proposed.  Thoughts about just using that?

**Comment by munsterlander** [comment]
Ok, I put an alternative up.

File: ethical_statement.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Preamble
+
+OpenDataCam (ODC) was created as an open-source tool to quantify the world. It is designed to be an accessible, affordable and open-source solution to better understand interactions in urban environments.
```
Comment (vsaw) [inline ethical_statement.md:3]: I think we should put this broader. ODC was also used to count fish (which I personally believe is a perfectly all right use case)

So maybe follow the statement of the website around "quantifying the world"?

File: ethical_statement.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+To that end, ODC has created a governance structure through which users should utilize in order to abide by the ODC ethical code. This document serves to define this code of ethics, with the objectives of maintaining integrity and trustworthiness through:
+
+*	Protection of the public interest
+*	Demonstration of personal integrity
+*	Showing respect for collaborators
+
+The ODC community is an open, safe and inclusive space fostering sharing and collaboration regardless of age, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual orientation. 
+
+This Code of Conduct applies as a condition of use but we expect everyone in our community to follow it.
```
Comment (vsaw) [inline ethical_statement.md:13]: I'm not a lawyer but this sentence makes it sound like it's part of the End User License Agreement, which as far as I understand is not what has been discussed in https://github.com/opendatacam/opendatacam/issues/390#issuecomment-842417970

> After taking a few days to reflect and spoke with @b-g also, decided to just include in a prominent way something like a "artist statement" on the website / documentation of OpenDataCam for next release... no re licensing, no code of conduct.. keep it very simple as always..

**Review by vsaw — CHANGES_REQUESTED** [review]
I stopped reading after [the following line](https://github.com/opendatacam/opendatacam/pull/488/files#diff-246ee8afe584281249b860826a12adc0f237abf2243d3237b28f5983c427fccaR13) as I see this is not what as intended by @tdurand and @b-g

**Review by munsterlander — COMMENTED** [review]
Works for me.  I was just trying to create a starting point of discussion so 4.0.0 work wouldn't all be left to you guys.

**Pull request merged**